core/src/main/java/com/graphhopper/routing/util/AbstractFlagEncoder.java

```diff
@@ -26,6 +26,7 @@
 import com.graphhopper.util.*;
 
 import java.util.*;
+import javax.xml.datatype.*;
```

Comment:
Hmmh, we cannot rely on this xml stuff in the encoder, this won't work for Android and iOS. Isn't it possible with something else or by hand where we support just the majority of possibilities?